### ISSUE TITLE
Map entrypoint property to cmd property

### ISSUE THREAD

**Issue by jzaefferer** [sloppyio/sloppose#9]
Map
```
entrypoint: /code/entrypoint.sh
```
to
```
cmd: /code/entrypoint.sh
```

See also https://docs.docker.com/compose/compose-file/#entrypoint

**Comment by malud** [comment]
and what happens with existing commands?
```
services:
  foo:
    image: busybox
    command: 5
    entrypoint: sleep
```

**Comment by jzaefferer** [comment]
I guess we can map both `entrypoint` and `command` to `cmd`. If both are specified, concat them with a space inbetween.

**Comment by jzaefferer** [comment]
Fixed by #19
